doc/user/content/cloud/administer-workspace.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Administer a Workspace"
+description: "Add or remove users to a Materialize Cloud workspace and set up workspace security."
+disable_toc: true
+menu:
+  main:
+    parent: "cloud"
+    weight:
+aliases:
+  - edit-deployments
+---
+
+{{< cloud-notice >}}
+
+Administrative users can set up shared workspaces and add and remove other users.
+
+To change a deployment name or size, click on the dots menu and select **Edit**.
+
+To add users or remove users, click on the profile icon in the top navbar and select **Settings**.
```

Comment:
s/Settings/Account settings/.

Reply:
One thing I couldn't figure out was how to "add and remove users" (assuming I'm the admin of my own account!).

Reply:
Oh, yeah, we just haven’t enabled this part yet. It’ll be in account settings when we turn it on.